### ISSUE TITLE
flaky test: test_operator_gpu.test_bilinear_sampler

### ISSUE THREAD

**Issue by lebeg** [apache/incubator-mxnet#12248]
http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/master/1456/pipeline

```
======================================================================
FAIL: test_operator_gpu.test_bilinear_sampler
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/local/lib/python3.5/dist-packages/nose/util.py", line 620, in newfunc
    return func(*arg, **kw)
  File "/work/mxnet/tests/python/gpu/../unittest/test_operator.py", line 4086, in test_bilinear_sampler
    assert_almost_equal(exe.grad_dict['grid'].asnumpy(), grid_grad, rtol=1e-3, atol=1e-5)
  File "/work/mxnet/python/mxnet/test_utils.py", line 491, in assert_almost_equal
    raise AssertionError(msg)
AssertionError: 
Items are not equal:
Error 633.610779 exceeds tolerance rtol=0.001000, atol=0.000010.  Location of maximum error:(0, 1, 32, 24), a=0.007807, b=0.038601
 a: array([[[[ 0.        , -0.0205371 ,  0.        , ...,  0.        ,
           0.        ,  0.        ],
         [ 0.        ,  0.        ,  0.        , ...,  0.        ,...
 b: array([[[[ 0.        , -0.0205371 ,  0.        , ...,  0.        ,
           0.        ,  0.        ],
         [ 0.        ,  0.        ,  0.        , ...,  0.        ,...
----------------------------------------------------------------------
```

**Comment by ankkhedia** [comment]
Hi @lebeg Thanks for filing the issue. We will look into the test.

**Comment by haojin2** [comment]
Fix in #12427
